get-started/setup-an-ide.md

```diff
@@ -1,7 +1,89 @@
 # Setup an IDE
 
-* vs code
-* goland
-* vim \(optional\)
-* emacs \(optional\)
+## vs code
 
+## GoLand
+
+You can use GoLand to easily run or debug TiDB in many situations.
+
+> Goland uses [delve](https://github.com/go-delve/delve) to debug go programs, usually GoLand has an usable delve and needn't any additional configuration. But in some cases, you may need to download a delve by yourself.
+
+0. Make sure your `go` environment work well, see previous sections for more information.
+1. Open TiDB project in GoLand.
+2. Add these config files to `<tidb-dir>/.idea/runConfigurations/` directory:
+
+  - unistore_4000.xml
+  ```xml
+  <component name="ProjectRunConfigurationManager">
+    <configuration default="false" name="unistore 4000" type="GoApplicationRunConfiguration" factoryName="Go Application">
+      <module name="tidb" />
+      <working_directory value="$PROJECT_DIR$" />
```

Comment:
does the audience need to replace this variable with the actual path?

Reply:
they needn't, this is GoLand's variable.

Reply:
they needn't because this is GoLand's variable, and has been tested for another macOS computer.